analysis/topEFT/topeft.py

```diff
@@ -122,10 +122,8 @@ def process(self, events):
         xsec               = self._samples[dataset]["xsec"]
         sow                = self._samples[dataset]["nSumOfWeights"]
 
-        # Turn systematics off if this is a data sample, otherwise get up down weights from input dict
-        if isData:
-            self._do_systematics = False
-        else:
```

Comment:
Just making sure, this was removed because now we _do_ have data systematics (`FFUp/Down`)? Did we double check the other systemics are still only being applied to the MC?

Reply:
Yes, the data numbers remain consistent with what we have on the master branch, so I think we should be good. I've also added an extra check [here](https://github.com/TopEFT/topcoffea/blob/745c44e455f143eb58d49eedb32a310f935577d8/analysis/topEFT/topeft.py#L780-L781) that should raise an error if any other up/down systematics are applied to data (this is in case we make some change to the systematics in the future and e.g. forget to include an `if not isData` statement, this exception should notify us that we have accidentally applied a non-data systematic fluctuation to data).

Reply:
Great!